R/lazy-select-query.R

```diff
@@ -94,6 +94,42 @@ update_lazy_select <- function(select, vars) {
   select
 }
 
+is_lazy_select_query_simple <- function(x,
```

Comment:
I think a brief comment explaining what "trivial" means here would be useful

Reply:
I added a comment and renamed `trivial` to `identity` which feels a bit clearer.